specs/latest/2.0/index.html

```diff
@@ -3117,6 +3117,18 @@ <h2 class="no-toc">Table of contents</h2>
         implementations.
     </div>
 
+    <h3>Framebuffer contents after invalidation</h3>
+    <p>
+        In OpenGL ES 3.0, after calling <code>invalidateFramebuffer</code> or <code>invalidateSubFramebuffer</code>,
+        the affected region's contents are undefined. In WebGL 2.0, it is required for the contents to either stay
+        unchanged or become all 0.
```

Comment:
Change to "... unchanged, reflect the results of subsequent rendering operations or become all 0.

Add a note:

As the immediate effect of calls to  <code>invalidateFramebuffer</code> or <code>invalidateSubFramebuffer</code> in OpenGL ES is likely to at most result in setting a state bit that is checked during later rendering, tests of this requirement are encouraged generate rendering before checking region's contents.